ml4gw/transforms/snr_rescaler.py

```diff
@@ -12,16 +12,19 @@ class SnrRescaler(FittableTransform):
     def __init__(
         self,
         n_ifos: int,
-        waveform_size: int,
-        highpass: float,
         sample_rate: float,
+        waveform_duration: float,
+        highpass: Optional[float] = None,
         distribution: Optional[Callable] = None,
     ):
+        super().__init__()
         self.distribution = distribution
```

Comment:
Would it make sense to include `target_snrs` as an argument of the `forward` method and relegate the sampling to a different `Module`?

Reply:
And I guess make it `Optional` so that obviously if it's `None` we do the SNR shuffling

Reply:
Agreed, cleaner and more moduler.